### ISSUE TITLE
Chat messages are sent in bulks when the player moves

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/hedgewars#546]
```
What steps will reproduce the problem?
1. Let the opponent spam multiple messages in online game while you move.

What is the expected output? What do you see instead?
Messages should be sent immediately after beeing writed. Instead they are sent 
in bulks.

What version of the product are you using? On what operating system?
hg tip 8579:90f445317e8a
Arch linux 64bit, opponent OS X

Please provide any additional information below.
I gave spaming as an error reproduce but it's noticable during normal 
conversation too.
```

Original issue reported on code.google.com by `ben...@unit22.org` on 28 Feb 2013 at 11:47

**Comment by GoogleCodeExporter** [comment]
```
Most likely due to unc0rr batching commands.  Presumably chat needs to be an 
exception.

```

Original comment by `kyberneticist@gmail.com` on 28 Feb 2013 at 11:51

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 6787f745da50.
```

Original comment by `unC0Rr` on 1 Mar 2013 at 8:16
* Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in r6787f745da50
```

Original comment by `unC0Rr` on 1 Mar 2013 at 8:17
